src/lib/viewers/media/DashViewer.js

```diff
@@ -245,6 +244,21 @@ class DashViewer extends VideoBaseViewer {
     }
 
     /**
+     * Handler for audio track
+     *
+     * @private
+     * @emits audiochange
+     * @return {void}
+     */
+    handleAudioTrack() {
+        const audioIdx = parseInt(this.cache.get('media-audiotracks'), 10);
```

Comment:
Is there a situation where the cache could be empty for `media-audiotracks`?

Reply:
There should be no such situation, because handleAudioTrack() is only ever called after selection in Settings.js, which sets the cache as the only means of communicating to this file what audio track should be switched to.